code/modules/clothing/under/syndicate.dm

```diff
@@ -12,6 +12,7 @@
 		laser = ARMOR_LASER_MINOR
 		)
 	siemens_coefficient = 0.9
+	material = /decl/material/solid/cloth
```

Comment:
Since these are antag items, these in particular probably don't need origin tech.

Reply:
Yeah since ANTAGS aren't a think I added em to the server. Or atleast ive been told they aren't gonna be a thing passed being event spawned in